frontend/public/components/catalog/catalog-items.tsx

```diff
@@ -93,15 +146,16 @@ export const catalogCategories = {
   },
 };
 
-const pageDescription =
+const pageDescription: string =
```

Comment:
You don't need to assign a type here. We can infer it.

Reply:
removed